draft-ietf-ppm-dap.md

```diff
@@ -2532,6 +2512,30 @@ different ways to address Sybil attacks, such as:
    {{anon-proxy}}.
 1. Differential privacy ({{dp}}) can help mitigate Sybil attacks to some extent.
 
+## Batch-selection Attacks {#batch-selection}
+
+Depending on the batch mode, the privacy of an individual Client may be
+infringed upon by selection of the batch. For example, in the leader-selected
```

Comment:
I went with a different change: everywhere we are discussing the modes in text, rather than specifying a specific enum variant, I use time-interval/leader-selected instead of `time_interval`/`leader_selected`. I think this reads a little better.